src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -55,7 +55,7 @@ public Climber() {
      * @return true if setting speed was successful
      */
     public boolean setSpeed(double spd) {
-        if (spd > 0 && ratchetEngaged) return false;
+        if (spd > 0 && (ratchetEngaged || climberRight.getEncoder().getPosition() > 80 || climberLeft.getEncoder().getPosition() > 80)) return false;
```

Comment:
this is still wrong, if any one of the encoders is above 80 none of them will move.